src/foraging_gui/Foraging.py

```diff
@@ -2205,6 +2205,7 @@ def _StartTrialLoop1(self,GeneratedTrials,worker1,workerPlot,workerGenerateAtria
             if self.ANewTrial==1 and self.ToGenerateATrial==1 and self.Start.isChecked(): 
                 self.ANewTrial=0 # can start a new trial when we receive the trial end signal from Bonsai
                 GeneratedTrials.B_CurrentTrialN+=1
+                print('Current trial: '+str(GeneratedTrials.B_CurrentTrialN+1))
```

Comment:
Why have the message about current trial in the terminal and in the log? I'm open with having it replicated, but I'm wondering what the motivation is

Reply:
Yes! Use for debugging or testing when you want to see an immediate response.